src/storage/store/rels_statistics.cpp

```diff
@@ -48,17 +51,17 @@ void RelsStatistics::assertNumRelsIsSound(
 }
 
 // We should only call this function after we call incrementNumRelsPerDirectionBoundTableByOne.
-void RelsStatistics::incrementNumRelsByOneForTable(table_id_t relTableID) {
+void RelsStatistics::updateNumRelsByValueForTable(table_id_t relTableID, int64_t value) {
     lock_t lck{mtx};
     initTableStatisticPerTableForWriteTrxIfNecessary();
     lck.unlock();
     setNumRelsForTable(relTableID,
         tablesStatisticsContentForWriteTrx->tableStatisticPerTable.at(relTableID)->getNumTuples() +
-            1);
+            value);
 }
 
-void RelsStatistics::incrementNumRelsPerDirectionBoundTableByOne(
-    table_id_t relTableID, table_id_t srcTableID, table_id_t dstTableID) {
+void RelsStatistics::updateNumRelsPerDirectionBoundTableByValue(
```

Comment:
Is updateNumRelsByValueForTable ever called without calling updateNumRelsPerDirectionBoundTableByValue? The comment on updateNumRelsByValueForTable makes it sound like the answer is no. If my understanding is right, we should just do all the work with 1 function call: updateNumRelsPerDirectionBoundTableByValue (and maybe just rename this to updateNumRels but keep the same arguments).

Reply:
yes, i created a new function:
```
void RelsStatistics::updateNumRelsByValue(
    table_id_t relTableID, table_id_t srcTableID, table_id_t dstTableID, int64_t value)
```
Which both update `mumRels` and `numRelsPerDirectionBoundTable`